src/app/components/nodes-view/nodes-view.component.ts

```diff
@@ -186,6 +187,20 @@ export class NodesViewComponent implements OnInit {
     return this.allocDataSource.data && this.allocDataSource.data.length === 0;
   }
 
+  formatColumn(){
+    if(this.nodeDataSource.data.length==0){
+      return
+    }
+    this.nodeColumnIds.forEach((colId)=>{
+      let emptyCell=this.nodeDataSource.data.filter((node)=>{
+        return (node as any)[colId]==="" || (node as any)[colId]==="n/a"
+      })
+      if (emptyCell.length==this.nodeDataSource.data.length){
```

Comment:
Hi @wusamzong 
Thanks for your reply, in line 191, and line 198 still have some compare `==` need to change to `===`.

Reply:
Thanks for your review; I have updated the code.